blocks/cards/cards.js

```diff
@@ -18,14 +20,21 @@ export async function fetchHtml(path) {
   return text;
 }
 
-export function renderCard(post) {
+async function extractTopicBasedOnLocale(topic) {
+  if (topic === undefined) return topic;
+  const apiResponse = await fetchAPI(`${TAGS_QUERY_INDEX}?sheet=topic`);
```

Comment:
Doing a fetch for every card, may be inefficient, even if the request might be cached on the disk by the browser.
Can you please save this in memory, following this pattern? Save the response of the first time you call the API on `window.blogTags` and if it's already present simply return it.

https://github.com/hlxsites/servicenow/blob/main/scripts/scripts.js#L138-L156
https://github.com/hlxsites/servicenow/blob/main/blocks/blogheader/blogheader.js#L21-L32
https://github.com/hlxsites/servicenow/blob/main/scripts/aem.js#L470-L499

if you look now in your networking tab, you will see 14 requests to the tags endpoint, although they all get the same information.

Reply:
You could use `ffetch` since @benpeter introduced it and please make sure you treat the error case too.